tests/AutoReview/FixerFactoryTest.php

```diff
@@ -62,7 +62,6 @@ public function provideFixersPriorityCases()
             [$fixers['array_indentation'], $fixers['binary_operator_spaces']],
             [$fixers['array_syntax'], $fixers['binary_operator_spaces']],
             [$fixers['array_syntax'], $fixers['ternary_operator_spaces']],
-            [$fixers['backtick_to_shell_exec'], $fixers['escape_implicit_backslashes']],
```

Comment:
could you elaborate on this one? are we sure there is no prioritiy conflict between them ?

I assume you didn't saw warning from https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/2.11/tests/IntegrationTest.php#L71 
(as if you would and willingly drop the test here, you would also drop the expection there)

Reply:
Also, `backtick_to_shell_exec` has
> // Should run before escape_implicit_backslashes

comment in `getPriority`

Reply:
on second thought I revered the change, making it more easy to pass this PR